css/styles.css

```diff
@@ -1,3 +1,59 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
+
 body {
+  background-color: black;
+  color: white;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
   font-size: 62.5%; /* Causes base font size to be ~10px */
+  font-family: Poppins, monospace;
 }
+
+.text-blurb {
+  width: 355px;
```

Comment:
Few notes about setting the width here and other places. 

1. I see you are setting this width on 3 different elements. A better way to do this could be wrapping those three elements in a div and setting the width on that div. I typically give this div a class name like container or something.
2. Instead of setting width I would use `max-width` to achieve this. The reason is that when you set width to a static number it's not very responsive. If someone with a screen smaller than 355px looks at this this element is going to go past the boundary of their screen. By using max-width it ensures that this is 355px wide on wide screens but on smaller screens it will shrink to fit them.